metrics.md

```diff
@@ -643,37 +643,47 @@ otherwise finding patterns in resource lifecycles.
-| Condition          | category         | http://terminology.hl7.org/CodeSystem/condition-category                  |
-| Device             | NA               | NA                                                                        |
-| DiagnosticReport   | NA               | http://loinc.org and<br>http://terminology.hl7.org/CodeSystem/v2-0074     |
-| DocumentReference  | category         | http://hl7.org/fhir/us/core/CodeSystem/us-core-documentreference-category |
-| Encounter          | type             | http://snomed.info/sct and<br>http://www.ama-assn.org/go/cpt              |
-| Immunization       | NA               | NA                                                                        |
-| Medication         | NA               | NA                                                                        |
-| MedicationRequest  | category         | http://terminology.hl7.org/CodeSystem/medicationrequest-category          |
-| Observation        | category         | http://terminology.hl7.org/CodeSystem/observation-category                |
-| Patient            | NA               | NA                                                                        |
-| Procedure          | NA               | NA                                                                        |
+Some systems below have codes in parentheses after the system URL.
+That means that you need only look for that specific code in the system.
+For example, if a `Condition.category` has a SNOMED system but a `12345678` code,
+you can treat it as an unrecognized system & code.
+
+| Resource Type      | Category Element | Category System                                                                                                                                                                           |
+|--------------------|------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| AllergyIntolerance | category         | `biologic`, `environment`, `food`, `medication`                                                                                                                                           |
+| Condition          | category         | http://hl7.org/fhir/us/core/CodeSystem/condition-category (`health-concern`) and <br>http://terminology.hl7.org/CodeSystem/condition-category and <br>http://snomed.info/sct (`16100001`) |
+| Device             | NA               | NA                                                                                                                                                                                        |
+| DiagnosticReport   | category         | http://loinc.org (`LP29684-5`, `LP29708-2`, `LP7839-6`) and<br>http://terminology.hl7.org/CodeSystem/v2-0074 (`LAB`)                                                                      |
```

Comment:
After we realized we messed up and left off some codes in Condition, I went through the other US Core profiles to see if we had done something similar.

All the rest of them seemed reasonable (either allowing all codes in a system, or a huge swath of them that would be impractical to constrain). Except that I realized the two profiles for DiagnosticReport did do a bit of constraining.

The Lab profile only cares about the `LAB` code. And the Note profile only lists 3 specific LOINC codes in its valueset. So I figured we ought to be similarly strict here? But I'm open to the argument that because they extensible, we shouldn't be so mean here - but I suppose that same argument could be made for SNOMED and Condition.category, which doesn't feel as good. I could go either way here.

Reply:
It looks like the lab and notes category valuesets becomes required in future versions of US Core, which to me would argue for being strict about them in the metrics, so I think this is right.